src/Microcosm.js

```diff
@@ -75,7 +74,7 @@ Microcosm.prototype = {
    */
   clean(transaction) {
     if (this.shouldRejectTransaction(transaction)) {
-      this.transactions = this.transactions.filter(t => t !== transaction)
+      this.transactions.splice(this.transactions.indexOf(transaction), 1)
```

Comment:
Small performance thing. This helps quite a bit when displaying high frequency data visualizations on mobile